<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="fileListWidget"/>
    </item>
    <item>
     <widget class="QPushButton" name="scanButton">
      <property name="text">
       <string>Scanner</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cleanButton">
      <property name="text">
       <string>Nettoyer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
